values.go

```diff
@@ -38,6 +39,12 @@ func GetFlags(dir string) (map[string]string, error) {
 
 	if version, err := versionFromFile(dir); err != nil {
 		return nil, fmt.Errorf("failed to get version: %v", err)
+	} else if version == "" {
+		// ex: v0.1.2 0.1.2 0.1.2.dev3
+		matched, err := regexp.MatchString(`^v?(\d+\.){1,2}(\d+)`, gitSummary)
```

Comment:
That would be preferable. Otherwise suddenly when somebody starts tagging their releases like 0.8.0-beta1 this will start giving empty strings again. I know this is how semver looks like but people do crazy versioning schemas all the time.

Summary() is always guaranteed to return some string even though there are no tags.